src/lib/editor.component.ts

```diff
@@ -119,6 +119,8 @@ export class NgxEditorComponent implements ControlValueAccessor, OnInit, OnChang
       subscription.unsubscribe();
     });
 
-    this.editor.destroy();
+    if (this.editor) {
+      this.editor.destroy();
```

Comment:
This line can be removed, this should be handled by wherever the Editor instance is created.